### PR TITLE
Google Analytics integrated

### PR DESCRIPTION
**Pull request by webmods-croatia**
https://github.com/mindstellar/Osclass/issues/211 part 1

### CODE REVIEW

**Comment by navjottomer** [comment]
Fix the conflict

**Comment by webmods-croatia** [comment]
> Fix the conflict

done

File: oc-admin/stats.php [hunk 1]
```diff
@@ -299,6 +299,23 @@ public function doModel()
...
+                    osc_add_flash_warning_message(_m("This action can't be done because it's a demo site"), 'admin');
+                    $this->redirectTo(osc_admin_base_url(true) . '?page=settings&action=advanced');
+                }
+                osc_csrf_check();
+
+                if (osc_set_preference('ganalytics_id', Params::getParam('tracking_id'))) {
+                    osc_add_flash_ok_message(_m('Google Analytics settings have been updated'), 'admin');
+                }
+                $this->redirectTo(osc_admin_base_url(true) . '?page=stats&action=google');
+                break;
```
Comment (navjottomer) [inline oc-admin/stats.php:317]: Two breakpoint.

Comment (navjottomer) [reply oc-admin/stats.php:317]: @webmods-croatia Please look into it.

**Comment by webmods-croatia** [comment]
@navjottomer Sorry, I'm a bit confused with this review thing, I made a new commit fixing it.

**Pull request merged**